scikitplot/plotters.py

```diff
@@ -92,6 +146,26 @@ def plot_confusion_matrix(y_true, y_pred, labels=None, title=None, normalize=Fal
         cm = np.around(cm, decimals=2)
         cm[np.isnan(cm)] = 0.0
 
+    if true_labels is None:
+        true_classes = classes
+    else:
+        validate_labels(classes, true_labels, "true_labels")
+
+        true_label_indexes = np.where(np.in1d(classes, true_labels))
```

Comment:
Can't say for sure, but I did some tests and wouldn't 
`true_label_indexes = np.in1d(classes, true_labels)`
work just as well?